packages/cli-test/src/cli/cli-process.ts

```diff
@@ -144,6 +146,9 @@ export class SlackCLIProcess {
       if (opts.token) {
         cmd = cmd.concat(['--token', opts.token]);
       }
+      if (opts.verbose) {
+        cmd = cmd.concat(['--verbose']);
```

Comment:
Using `concat` alines with the syntax of this function and makes it readable, but looking at this line alone made me ask why `push` was not used instead 🤔

Reply:
IIRC `push` and `concat` are doing similar things but with different returns and underlying happenings - instead of adding to the existing array, `concat` creates a new one and returns this as a value while `push` returns the new length instead 📚 ✨

FWIW I think `push` is a reasonable substitution, but `concat` gives me confidence in fewer side effects.